cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamFileJanitorService.java

```diff
@@ -68,7 +68,6 @@ public void run() {
           LOG.debug("Completed stream file cleanup.");
         } catch (Throwable e) {
           LOG.warn("Failed to cleanup stream file: {}", e.getMessage());
-          LOG.debug("Failed to cleanup stream file.", e);
```

Comment:
warn is a higher log level, so the message logged by the `LOG#debug` statement would be redundant if LOG level is set to show debugs.